diff --git a/mathlive.js b/mathlive.js
index 1f8aed7..034840a 100644
--- a/mathlive.js
+++ b/mathlive.js
@@ -39041,9 +39041,9 @@ data-command='["setEnvironment","pmatrix"]'>
     if (offset < first || offset > last) return false;
     if (!atom.hasChildren) return true;
     const firstChild = includeFirstAtoms ? atom.firstChild : firstNonFirstChild(atom);
-    if (!firstChild) return false;
+    if (!firstChild) return true;
     const lastChild = includeFirstAtoms ? atom.lastChild : lastNonFirstChild(atom);
-    if (!lastChild) return false;
+    if (!lastChild) return true;
     const firstOffset = model.offsetOf(firstChild);
     if (firstOffset >= first && firstOffset <= last) {
       const lastOffset = model.offsetOf(lastChild);
diff --git a/mathlive.mjs b/mathlive.mjs
index 1748b47..774270f 100644
--- a/mathlive.mjs
+++ b/mathlive.mjs
@@ -39001,9 +39001,9 @@ function atomIsInRange(model, atom, first, last, includeFirstAtoms) {
   if (offset < first || offset > last) return false;
   if (!atom.hasChildren) return true;
   const firstChild = includeFirstAtoms ? atom.firstChild : firstNonFirstChild(atom);
-  if (!firstChild) return false;
+  if (!firstChild) return true;
   const lastChild = includeFirstAtoms ? atom.lastChild : lastNonFirstChild(atom);
-  if (!lastChild) return false;
+  if (!lastChild) return true;
   const firstOffset = model.offsetOf(firstChild);
   if (firstOffset >= first && firstOffset <= last) {
     const lastOffset = model.offsetOf(lastChild);
diff --git a/package.json b/package.json
index 5bdefaa..8a45137 100644
--- a/package.json
+++ b/package.json
@@ -29,7 +29,7 @@
     "mathquill"
   ],
   "main": "./mathlive.min.js",
-  "module": "./mathlive.min.mjs",
+  "module": "./mathlive.mjs",
   "types": "./types/mathlive.d.ts",
   "exports": {
     "./vue": "./vue-mathlive.mjs",
@@ -39,8 +39,8 @@
       "browser": {
         "production": {
           "types": "./types/mathlive.d.ts",
-          "import": "./mathlive.min.mjs",
-          "require": "./mathlive.min.js"
+          "import": "./mathlive.mjs",
+          "require": "./mathlive.js"
         },
         "development": {
           "types": "./types/mathlive.d.ts",
@@ -54,8 +54,8 @@
       },
       "default": {
         "types": "./types/mathlive.d.ts",
-        "import": "./mathlive.min.mjs",
-        "require": "./mathlive.min.js"
+        "import": "./mathlive.mjs",
+        "require": "./mathlive.js"
       }
     },
     "./ssr": {
